### PR TITLE
[NetAppFiles] Bump to 2019-11-01 API and support to revert volume to a snapshot

### PR DESCRIPTION
**Pull request by audunn**
**Description**  

Updates NetAppFiles Azure CLI to API version 2019-01, adding Volume Revert operation used to revert a volume to one of its snapshots. Removes mount-targets operation as mount targets information is available as part of Volume properties.

**Testing Guide**  

```
// Create a snapshot from a volume 
az netappfiles snapshot create -g mygroup --account-name myaccname --pool-name mypoolname --name myvolname 
// Revert a volume to a snapshot 
az netappfiles volume revert -g mygroup --account-name myaccname --pool-name mypoolname --name myvolname --snapshot-id 9760acf5-4638-11e7-9bdb-020073ca3333

```
**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

[NetAppFiles] az volume revert:  Added Volume Revert to revert a volume to one of its snapshots.  
[NetAppFiles] BREAKING CHANGE: az netappfiles mount-target: Removed.  
[NetAppFiles] az volume show: Added site to Active Directory Properties

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
NetAppFiles

**Comment by audunn** [comment]
@haroldrandom test recordings have been updated.
Can we please get this reviewed asap as customers are waiting for changes in this PR.

**Comment by audunn** [comment]
@haroldrandom  All tests run successfully I don't see why the pipeline should be failing now.  Can you advice ?

File: .vscode/launch.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
             "name": "Azure CLI Debug (Integrated Console)",
             "type": "python",
             "request": "launch",
-            "pythonPath": "${config:python.pythonPath}",
+            "pythonPath": "${config:python.interpreterPath}",
```
Comment (haroldrandom) [inline .vscode/launch.json:8]:
```suggestion
            "pythonPath": "${config:python.pythonPath}",
```

File: .vscode/launch.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
             "type": "python",
             "request": "launch",
             "stopOnEntry": true,
-            "pythonPath": "${config:python.pythonPath}",
+            "pythonPath": "${config:python.interpreterPath}",
```
Comment (haroldrandom) [inline .vscode/launch.json:27]:
```suggestion
            "pythonPath": "${config:python.pythonPath}",
```

File: .vscode/launch.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
             "name": "Azdev Scripts",
             "type": "python",
             "request": "launch",
-            "pythonPath": "${config:python.pythonPath}",
+            "pythonPath": "${config:python.interpreterPath}",
```
Comment (haroldrandom) [inline .vscode/launch.json:43]:
```suggestion
            "pythonPath": "${config:python.pythonPath}",
```

**Pull request merged**